### PR TITLE
Added release notes for 2.18

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md [hunk 1]
```diff
@@ -6,10 +6,96 @@ categories: releasenotes
...
 
-## Bla
-Bla
+## Improved Java 11 Support
+
+Xtext's support for Java 11 has been further improved:
+
+* Improved annotation processing in Xbase. ([xtext-extras#425](https://github.com/eclipse/xtext-extras/issues/425), [xtext-xtend#800](https://github.com/eclipse/xtext-xtend/issues/800))
+* Several features are relying on querying the classpath, e.g. content assist. ([xtext-web#128](https://github.com/eclipse/xtext-web/issues/128))
+* New Java 11 API methods broke test assumptions when running on a Java 11 VM. All affected test cases have been refactored. ([xtext-extras#424](https://github.com/eclipse/xtext-extras/issues/424))
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md]: is the noteworthy?

File: xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md [hunk 2]
```diff
@@ -6,10 +6,96 @@ categories: releasenotes
...
+
+### Fixed Potential Deadlock on Startup
+
+A deadlock may occur on startup of Eclipse. The builder infrastructure has been improved to avoid locking on loading its stored state. ([#1460](https://github.com/eclipse/xtext/issues/1460))
+
+### Fixed: Exceptions for Closed Projects
+
+When projects are removed from the workspace the `ProjectOpenedOrClosedListener` may caused exceptions that were logged to the error log. This situation has been fixed with [xtext-eclipse#1066](https://github.com/eclipse/xtext-eclipse/issues/1066).
+
+### Refactoring Support: Improved Progress Monitoring
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md]: reporting?

File: xtend-website/_posts/releasenotes/2019-06-04-version-2-18-0.md [hunk 3]
```diff
@@ -46,6 +46,38 @@ def m() {
...
+
+### New Quickfix: Rename Class
+
+When an Xtend class name does not match the file name it is defined in then Xtend raises a warning. A new quickfix has been provided to rename the class according to the file name. ([#744](https://github.com/eclipse/xtext-xtend/issues/744))
+
+![Quickfix: Rename Class]({{site.baseurl}}/images/releasenotes/2_18_Quickfix_RenameClass.gif){:height="50%" width="50%"}
+
+### Quickfix "Remove Unused Member" Improved
+
+The quickfix "Remove member" for unused members was improved. Additionally to the member itselfs the quickfix will now also remove obsolete import statements write accesses. ([xtext-xtend](https://github.com/eclipse/xtext-xtend/issues/736))
```
Comment (miklossy) [inline xtend-website/_posts/releasenotes/2019-06-04-version-2-18-0.md]: and is missing: remove the obsolete import statements **and** the write accesses

File: xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md (hunk 1 above)
Comment (szarnekow) [reply xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md]: I don't think so.

File: xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md (hunk 2 above)
Comment (kthoms) [reply xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md]: "reporting" is better here, although cancellation is not really reporting

File: xtend-website/_posts/releasenotes/2019-06-04-version-2-18-0.md (hunk 3 above)
Comment (kthoms) [reply xtend-website/_posts/releasenotes/2019-06-04-version-2-18-0.md]: I don't think that adding *the* is correct here. Added *and* for next commit.

File: xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md (hunk 1 above)
Comment (kthoms) [reply xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md]: Removed the whole paragraph

**Comment by kthoms** [comment]
Updated with applied review comments. @szarnekow @cdietrich @miklossy Please check again.

**Pull request merged**